db_importer.py

```
@@ -0,0 +1,121 @@
+    Import data from the Genie VCF to the variant table.
+    
+    Returns
+    -------
+    None    
+    """
+    # Connet to the SQLite database.
+    db = sqlite3.connect(settings.DATABASES['default']['NAME'])
+    cur = db.cursor()
+    
+    # Delete all previous variant records.
+    sql_query = f'DELETE FROM main_variant'
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)